Product/Production/Services/CORE_X12DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/corex12/docsubmission/genericbatch/request/outbound/PassthroughOutboundCORE_X12DSGenericBatchRequest.java

```diff
@@ -73,9 +78,10 @@ public COREEnvelopeBatchSubmissionResponse batchSubmitTransaction(COREEnvelopeBa
         COREEnvelopeBatchSubmissionResponse oResponse = null;
         NhinTargetSystemType targetSystem = MessageGeneratorUtils.getInstance().convertFirstToNhinTargetSystemType(
             targets);
-        OutboundCORE_X12DSGenericBatchRequestOrchestratable dsOrchestratable
-            = createOrchestratable(dsDelegate, msg, targetSystem, assertion);
+        this.auditRequestToNhin(msg, assertion, targetSystem);
```

Comment:
This is the first place in this PR that we're using "this" directly; do we have a defined convention?